repositories/scenario_testrun.go

```diff
@@ -0,0 +1,81 @@
+package repositories
+
+import (
+	"context"
+	"time"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/repositories/dbmodels"
+)
+
+type ScenarioTestRunRepository interface {
+	CreateTestRun(ctx context.Context, tx Transaction, testrunID string,
+		input models.ScenarioTestRunInput) error
+	GetByScenarioIterationID(ctx context.Context, exec Executor, scenarioID string) (models.ScenarioTestRun, error)
+	GetByID(ctx context.Context, exec Executor, testrunID string) (models.ScenarioTestRun, error)
+}
+
+type ScenarioTestRunRepositoryPostgresql struct{}
```

Comment:
here as well we have a bit of a mix of two conventions in the code, but I'd rather push you in the direction of: define new methods on the struct `MarbleDbRepository` (see any one of the more "recent" repositories, for instance the case_repository.
This way we have one concrete implementation of the repository, and we can just use minimal interfaces at the usecase level

Reply:
(then you may have to make the method names more complete however)